vendor/github.com/openshift/library-go/pkg/operator/encryption/controllers/migrators/inprocess.go

```diff
@@ -127,15 +130,19 @@ func (m *InProcessMigrator) runMigration(gvr schema.GroupVersionResource, writeK
 	listPager := pager.New(pager.SimplePageFunc(func(opts metav1.ListOptions) (runtime.Object, error) {
 		for {
 			allResource, err := d.List(opts)
-			if err != nil && !canRetry(err) {
+			if err != nil {
 				klog.Warningf("List of %v failed: %v", gvr, err)
```

Comment:
make it `klog.V(4)` ?

Reply:
Isn't this worth a warning? It's once per resource. We return the error and wait 5 min.